### PR TITLE
fix align_corners doc

### PR DESCRIPTION
**Pull request by ssnl**

### CODE REVIEW

**Comment by SsnL** [comment]
cc @bnehoran

File: torch/nn/functional.py [hunk 1]
```diff
@@ -2362,12 +2362,13 @@ def upsample(input, size=None, scale_factor=None, mode='nearest', align_corners=
...
-            output tensors are aligned by the corner points of their corner
-            pixels, and the interpolation uses edge value padding for out-of-boundary values.
-            This only has effect when :attr:`mode` is ``'linear'``,
-            ``'bilinear'``, ``'bicubic'`` or ``'trilinear'``.
+            If set to ``True``, the input and output tensors are aligned by the
+            center points of their corner pixels, preserving the values at the corner pixels.
+            If set to ``False``, the input and output tensors are aligned by the corner
+            points of their corner pixels, and the interpolation uses edge value padding
+            for out-of-boundary values, making this operation *independent* of input size
+            when :attr:`scale_factor` is kept the same. This only has effect when :attr:`mode`
```
Comment (bnehoran) [inline torch/nn/functional.py]:
```suggestion
            when :attr:`scale_factor` is kept the same. This only has an effect when :attr:`mode`
```

File: torch/nn/functional.py [hunk 2]
```diff
@@ -2415,12 +2416,13 @@ def interpolate(input, size=None, scale_factor=None, mode='nearest', align_corne
...
-            output tensors are aligned by the corner points of their corner
-            pixels, and the interpolation uses edge value padding for out-of-boundary values.
-            This only has effect when :attr:`mode` is ``'linear'``,
-            ``'bilinear'``, ``'bicubic'``, or ``'trilinear'``.
+            If set to ``True``, the input and output tensors are aligned by the
+            center points of their corner pixels, preserving the values at the corner pixels.
+            If set to ``False``, the input and output tensors are aligned by the corner
+            points of their corner pixels, and the interpolation uses edge value padding
+            for out-of-boundary values, making this operation *independent* of input size
+            when :attr:`scale_factor` is kept the same. This only has effect when :attr:`mode`
```
Comment (bnehoran) [inline torch/nn/functional.py]:
```suggestion
            when :attr:`scale_factor` is kept the same. This only has an effect when :attr:`mode`
```

**Review by bnehoran — APPROVED** [review]
Just a minor grammatical change if we're already modifying the line. Really not important.
Otherwise looks good.

**Comment by SsnL** [comment]
@bnehoran Thanks!

@pytorchbot merge this please

**Comment by bnehoran** [comment]
@SsnL I think you may have the wrong pull request number listed at the bottom of https://github.com/pytorch/pytorch/issues/23555#issuecomment-517526640

**Comment by SsnL** [comment]
@bnehoran fixed!

**Comment by vincentqb** [comment]
This looks good to me. I'll land.

**Pull request closed without merging**